datastream-server/src/test/java/com/linkedin/datastream/server/assignment/TestStickyPartitionAssignment.java

```diff
@@ -704,20 +695,29 @@ public void testClusterNameWithElasticTaskEnabledConfigMismatch() {
     int fullnessFactorPct = 50;
 
     // Pass null or blank clusterName while elastic task assignment is enabled. This should throw.
-    Assert.assertThrows(IllegalArgumentException.class, () -> new StickyPartitionAssignmentStrategy(Optional.empty(),
-        Optional.empty(), Optional.empty(), true, Optional.of(partitionsPerTask), Optional.of(fullnessFactorPct),
-        Optional.of(_zkClient), null));
+    Assert.assertThrows(IllegalArgumentException.class, () -> createStickyPartitionAssignmentStrategy(partitionsPerTask,
+        fullnessFactorPct, true, _zkClient, null));
 
-    Assert.assertThrows(IllegalArgumentException.class, () -> new StickyPartitionAssignmentStrategy(Optional.empty(),
-        Optional.empty(), Optional.empty(), true, Optional.of(partitionsPerTask), Optional.of(fullnessFactorPct),
-        Optional.of(_zkClient), ""));
+    Assert.assertThrows(IllegalArgumentException.class, () -> createStickyPartitionAssignmentStrategy(partitionsPerTask,
+        fullnessFactorPct, true, _zkClient, ""));
 
     // If elastic task assignment is disabled, it should not matter whether clusterName is null or has a valid value.
-    new StickyPartitionAssignmentStrategy(Optional.empty(), Optional.empty(), Optional.empty(), false,
-        Optional.empty(), Optional.empty(), Optional.empty(), null);
+    createStickyPartitionAssignmentStrategyObject(0, 0, null, null);
+
+    createStickyPartitionAssignmentStrategyObject(0, 0, null, _clusterName);
+  }
+
+  private void createStickyPartitionAssignmentStrategyObject(int partitionsPerTask, int partitionFullnessFactorPct,
```

Comment:
nit: rather than adding this extra function, can't you just directly call `createStickyPartitionAssignmentStrategy` with "enableElasticTask" set to 'false' and just ignore the returned assignment?

Reply:
Created a default to avoid modifying some of the defaults everywhere. It was easier to refactor and fix at one place.

Reply:
nit: can you rename this function to either be the same as `createStickyPartitionAssignmentStrategy` or `createStickyPartitionAssignmentStrategyWithElasticTaskDisabled`?